daemon/kmd/api/swagger.json

```diff
@@ -644,7 +644,7 @@
       "x-go-package": "github.com/algorand/go-algorand/daemon/kmd/lib/kmdapi"
     },
     "APIV1POSTKeyExportResponse": {
-      "description": "APIV1POSTKeyExportResponse is the reponse to `POST /v1/key/export`\nfriendly:ExportKeyResponse",
+      "description": "APIV1POSTKeyExportResponse is the response to `POST /v1/key/export`\nfriendly:ExportKeyResponse",
```

Comment:
this is an auto-gen file, so you'll need to make the changes to the source files and re-run `make rebuild_swagger`

Reply:
I see, the linter fixed it in both places but the rebuild triggered a rebuild of the bundled JSON spec, will commit and add it.